README.md

```diff
@@ -13,7 +13,7 @@ See [Accessing Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developergui
 
 ### Required Golang versions
 
-qldbdriver 1.0.0 requires Golang 1.14 or later.
+QldbDriver 1.0.1 requires Golang 1.14 or later.
```

Comment:
This should be `QldbDriver 1.x requires Golang 1.14 or later`. Both, v1.0.0 and v1.0.1, and all the later upcoming versions of 1.x will require Golang 1.14 or later

Reply:
Updated